style/theme.ts

```diff
@@ -94,6 +94,77 @@ const theme = createTheme(base, {
         },
       },
     },
+    MuiCard: {
+      styleOverrides: {
+        root: {
+          display: 'flex',
+          boxShadow: '3px 5px 11px 3px rgba(0, 0, 0, 0.13)',
+
+          flexDirection: 'column',
+          [base.breakpoints.up('md')]: {
+            flexDirection: 'row',
+          },
+          borderRadius: '0.375rem',
+        },
+      },
+    },
+    MuiCardMedia: {
+      styleOverrides: {
+        root: {
+          width: '100%',
+          minHeight: '200px',
+          position: 'relative',
+          maxWidth: '100%',
+          backgroundColor: 'rgb(205, 180, 248)',
```

Comment:
not a fan of this color but if were going w a color can you add it to the palette